lib/state.ml

```diff
@@ -411,9 +414,11 @@ let epoch_of_session server peer_name protocol_version = function
     let epoch : epoch_data = common_data_to_epoch session.common_session_data13 server peer_name in
     {
       epoch with
+      protocol_version       = protocol_version ;
```

Comment:
while here, I thought it'd be nice to actually include the TLS_1_3 in the epoch ;) (read: this was a bug before, and is fixed now)